locations/spiders/carrabbasitaliangrill.py

```diff
@@ -7,68 +7,62 @@
 from locations.hours import OpeningHours
 
 
-DAY_MAPPING = {
-    'Monday': 'Mo',
-    'Tuesday': 'Tu',
-    'Wednesday': 'We',
-    'Thursday': 'Th',
-    'Friday': 'Fr',
-    'Saturday': 'Sa',
-    'Sunday': 'Su'
-}
-
 class CarrabbasItalianGrillSpider(scrapy.Spider):
     download_delay = 0.2
     name = "carrabbasitaliangrill"
     allowed_domains = ["carrabbas.com"]
     start_urls = (
-        'https://www.carrabbas.com/locations/all',
+        'https://locations.carrabbas.com/index.html',
     )
 
     def parse(self, response):
```

Comment:
A sitemap spider would remove the need to parse the state and city pages.

[sitemap](https://locations.carrabbas.com/sitemap.xml)
[example spider](https://github.com/alltheplaces/alltheplaces/blob/eb7b1683266c98a8102f7441ee5a9f20f5ab6121/locations/spiders/puregym.py)